meilisearch-release-changelog/v1.5.0.md

```diff
@@ -0,0 +1,89 @@
+# v1.5.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.5 introduces improvements in indexing speed and the possibility of triggering snapshots on-demand, as it's already possible with dumps.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Indexing speed improvements
+
+For datasets containing a lot of words (and/or with a large variety of words), we improved the indexing speed between 5% and 20%, depending on your use cases and your documents. The more your documents contain a lot of words, the more the improvements will be visible.
+
+The relevancy can be slightly different for queries containing 4 words or more. Let us know [here](https://github.com/meilisearch/product/discussions) if you have any issues or unexpected behavior.
+
+For datasets containing only a few words per field, even if you have a huge number of fields, the improvement might not be visible. Don't worry, we keep working on the indexing speed in the next sprint to address other use cases.
```

Comment:
> Do we have some sort of indicative number of what would be a lot of words for Meilisearch? A field with 10, 100, 1000 words? I think it would help a lot and make it easier for users to judge whether this is relevant to them.

To be 100% true, it's not a matter of number of words per field, but a number of different words in the datasets. So let's say at least a field with 100 words in their document, if it can help the users to have a number

> Also, should we already have a discussion prepared for this? Feels a bit weird to say "if you have problems/feedback, go to this page" when the linked page is a list of discussions unrelated to these performance improvements.

We do not want feedback about perf improvement, but about relevancy changes.
I created a product discussion -> https://github.com/orgs/meilisearch/discussions/704

Reply:
I updated @guimachiavelli, let me know

Reply:
made a few minor edits to your changes, but we should be good to go